Update 4 (#251)
NilPointer-Software/bdfd-wiki: src/bdscript/authorOfMessage.md

# $authorOfMessage
Returns the ID of the provided message's author.

## Syntax
```
$authorOfMessage[channel ID;message ID]
```

## Parameters

> - `channel ID`: The channel where the message was sent.
> - `message ID`: The message that the bot returns the author ID for.

Reviewer: ```suggestion
- `channel ID`: The channel where the message was sent in.
- `message ID`: The message for which the author ID will be returned.
```

> >  [How get message/channel ID guide.](https://support.discord.com/hc/en-us/articles/206346498-Where-can-I-find-my-User-Server-Message-ID-)

Reviewer: ```suggestion
>  [How to get the message/channel ID guide.](https://support.discord.com/hc/en-us/articles/206346498-Where-can-I-find-my-User-Server-Message-ID-)
```
